sdk/storage/azblob/blockblob/client.go

```diff
@@ -260,6 +269,11 @@ func (bb *Client) CommitBlockList(ctx context.Context, base64BlockIDs []string,
 			ImmutabilityPolicyExpiry:  options.ImmutabilityPolicyExpiryTime,
 		}
 
+		// If user attempts to pass in their own checksum, errors out.
+		if options.TransactionalValidation != nil && reflect.TypeOf(options.TransactionalValidation).Kind() != reflect.Func {
+			return CommitBlockListResponse{}, bloberror.UnsupportedChecksum
```

Comment:
Instead of returning error here, can we just log this (saying it is not supported) and go ahead with the operation?

Reply:
I think it's better to return an error so the customer doesn't think their data is validated when it wasn't